lib/miq_environment.rb

```diff
@@ -41,6 +41,10 @@ def self.manageiq_gid
   class Command
     EVM_KNOWN_COMMANDS = %w[apachectl docker nohup podman service systemctl].freeze
 
+    def self.arch
+      Sys::Platform::ARCH
+    end
```

Comment:
I think this should move up to MiqEnvironment as opposed to MiqEnvironment::Command.  Command is more meant for when we need to shell out to find information.

(That being said, I don't know why we have that distinction for Command at all)